tools/cli/src/opt/kanidm.rs

```diff
@@ -672,6 +698,34 @@ pub enum ServiceAccountOpt {
     IntoPerson(AccountNamedOpt),
 }
 
+#[derive(Debug, Subcommand)]
+pub enum ApplicationOpt {
+    /// List all applications
+    #[clap(name = "list")]
+    List(CommonOpt),
+    /// Create a new application
+    #[clap(name = "create")]
+    Create(Named),
+    /// Delete an application
+    #[clap(name = "delete")]
+    Delete(Named),
+    /// Add new members
+    #[clap(name = "add-members")]
+    AddMembers(GroupNamedMembers),
```

Comment:
Given we plan to just have the application associate a group which has the members, I think this cli interface may be a bit out of date.

Reply:
Yes. There will be a new one matching the design doc.

Reply:
As mentioned, you should probably focus on writing use case oriented tests next